examples/new_examples/new_examples.md

```diff
@@ -17,8 +17,8 @@ Can GitHub Actions run the notebooks and check for errors? (Treebeard?)
 3. "Good enough" found during initialisation
 4. batch vs. single opt.ask()
 5. Noise
-6. Narrow wells
-7. Explore vs. exploit (Gold Diggerson vs. Map Drawson)
```

Comment:
I think explore vs. exploit is the natural next place to go, since we have just talked about strategies.